qml/nemo-keyboard.qml

```diff
@@ -98,6 +129,50 @@ Item {
             color: Theme.backgroundColor
         }
 
+        Rectangle{
+            id: predictorView
+            width: keyboard.width
+            height: Theme.itemHeightLarge
+            color: Theme.backgroundColor
+
+            anchors.bottom: keyboard.top
+            visible: enablePredictor.value == true
```

Comment:
The enablePredictor.value is not boolean? Is there some reason to compare with true?